src/test/java/com/suse/saltstack/netapi/utils/ClientUtilsTest.java

```diff
@@ -63,5 +63,4 @@ public void testStreamToString() {
                 new ByteArrayInputStream(TEST_STRING.getBytes()));
         assertEquals("Result doesn't match test string", TEST_STRING, result);
     }
-
```

Comment:
Oops seems like a left over from a previous version i will remove it. (or rather add it back :smile: )